README.md

```diff
@@ -625,6 +625,39 @@ const uploadResult = await turbo.uploadFile({
 });
 ```
 
+##### Customize Multi-Part Upload Behavior
+
+By default, the Turbo upload methods will split files into chunks that are larger than 10 MiB and send these chunks to the upload service multi-part endpoints. This behavior can be customized with the following inputs:
+
+- `chunkByteCount`: The maximum size in bytes for each chunk. Must be between 5 MiB and 500 MiB. Defaults to 5 MiB.
+- `maxChunkConcurrency`: The maximum number of chunks to upload concurrently. Defaults to 5.
+- `chunkingMode`: The chunking mode to use. Can be 'auto', 'force', or 'disabled'. Defaults to 'auto'. Auto behavior means chunking is enabled if the file would be split into at least three chunks.
```

Comment:
suggestion: may be helpful to describe why you would change these

`maxChunkConcurrency` - reducing it will slow down uploads, but reduce memory utiliazation and serialize network calls. increasing it will be faster, but can strain available resources